executor/cover_filter.h

```diff
@@ -0,0 +1,139 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// CoverFilter is PC hash set that can be placed in shared memory.
+//
+// The set can cover up to 4 distinct 1GB regions of PCs.
```

Comment:
Will it work normally in the presence of kernel modules?

Judging by #4828, it's quite possible that modules are separated in the memory by more than `2^32` bytes. Do we hope that they will fit in only 4 distinct regions?

Reply:
In #4828 the diff between _stext and last module is just 1.2GB (0xffffffd3eeb2a000-0xffffffd2eeb2a000).
My assumption is that the worst case scenario is that vmlinux and modules are far from each other and both cross 1GB aligned boundary, but even in that case 4 regions is enough.